src/api/apiLayer.tsx

```diff
@@ -40,6 +40,16 @@ export async function getActivites(
     method: 'GET',
   })
     .then(async (res) => await res.json())
+    .then(({ activities }) =>
+      activities.map((activity: any) => ({
+        id: activity.id?.toString(),
+        title: activity.title,
+        startTime: formatDate(activity.startTime),
+        endTime: activity.endTime ?? '',
```

Comment:
bro why did you do 1 but not both 🥲

Reply:
I looked in the /addActivity and endTime was not there. I thought it was deleted so I did not do anything for it and forgot to ask why this was the case. I'll add it back 😦